floyd/cli/data.py

```diff
@@ -159,18 +161,30 @@ def delete(ids, yes):
             failures = True
             continue
 
-        if not yes and not click.confirm("Delete Data: {}?".format(data_source.name),
+        data_name = normalize_data_name(data_source.name)
+        suffix = data_name.split('/')[-1]
+        if not suffix.isdigit():
+            failures = True
+            floyd_logger.error('%s is not a dataset, skipped.', id)
```

Comment:
We should probably log this in the else section of the if condition following this.